MANIFEST.in

```
@@ -1,2 +1,3 @@
 include build.rs Cargo.toml Cargo.lock
 recursive-include src *
+recursive-include apple-catalog-parsing *
```

Ensure file ends in newline

```suggestion
recursive-include apple-catalog-parsing *

```